src/test/java/org/sagebionetworks/bridge/exporter3/Exporter3WorkerProcessorTest.java

```diff
@@ -833,6 +857,7 @@ private void verifySynapseExport(String expectedS3Key) throws Exception {
         assertEquals(DateTime.parse(flattenedAnnotationMap.get(Exporter3WorkerProcessor.METADATA_KEY_UPLOADED_ON)).getMillis(),
                 UPLOADED_ON_MILLIS);
         assertEquals(flattenedAnnotationMap.get(CUSTOM_METADATA_KEY_SANITIZED), CUSTOM_METADATA_VALUE_CLEAN);
+        assertEquals(flattenedAnnotationMap.get(Exporter3WorkerProcessor.METADATA_KEY_CONTENT_TYPE), CONTENT_TYPE);
```

Comment:
You will also need to add this to verifyS3Metadata() above.

Reply:
Good catch thanks